### ISSUE TITLE
override Record.description

### ISSUE THREAD

**Issue by mtissington** [groue/GRDB.swift#58]
This maybe an issues with my knowledge of Swift

```swift
class Group: Record  {
    var name: String
    var description: String
    var enabled: Bool
```

I get an error cannot override member variable (description) ... 
Is there a way around this short of using a different name?

**Comment by groue** [comment]
Hi @mtissington.

Today the Record class adopts the [CustomStringConvertible](https://developer.apple.com/library/watchos/documentation/Swift/Reference/Swift_CustomStringConvertible_Protocol/index.html) protocol, which defines the `description` property. Hence the conflict.

This was a bad move. It looked nice at first, because print(record) prints something nice. It has some defects, and you have just faced one of it. I should remove the default adoption of CustomStringConvertible. Consider it on the TODO list.

**Comment by groue** [comment]
Until I release the fix, use a different property name. Sorry for the inconvenience.

**Comment by groue** [comment]
The fix has shipped in v0.66.0.
